src/components/LanguageControl/index.js

```diff
@@ -10,6 +10,7 @@ const LanguageControl = ({ languages }) => {
   const handleChange = (event) => {
     const { value } = event.target;
     i18n.changeLanguage(value);
+    localStorage.setItem('lang', value);
```

Comment:
I'd prefer session storage in this case

Reply:
You can find the difference in this article https://learn.javascript.ru/localstorage